src/test/java/redis/clients/jedis/JedisClusterTest.java

```diff
@@ -780,4 +781,17 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     }
     return false;
   }
+
+  @Test
+  public void broadcast() {
+    try (JedisCluster cluster = new JedisCluster(Collections.singleton(new HostAndPort(LOCAL_IP, 7379)),
+        DefaultJedisClientConfig.builder().password("cluster").build())) {
+      JedisBroadcast broadcast = new JedisBroadcast(cluster);
+      
+      Map<?, Supplier<String>> replies = broadcast.broadcastCommand(new CommandObject<>(
```

Comment:
may be we should add `PING`JedisBoardcast.java ?

Reply:
@yangbodong22011 After we decided on a design, we can add many commands :slightly_smiling_face: